src/components/atomic/spinner/__tests__/Spinner.tests.js

```diff
@@ -1,14 +1,14 @@
-import { render } from '@testing-library/react';
+import { render, screen } from '@testing-library/react';
 import React from 'react';
 
 import Spinner from '../Spinner';
 
 describe('Spinner component', () => {
-	test('should have expected button label and fire onRemove handler', () => {
-		const { container } = render(<Spinner />);
-		expect(container.querySelector('.nci-spinner')).toBeInTheDocument();
-		expect(container.querySelector('.spinkit')).toBeInTheDocument();
-		expect(container.querySelector('.dot1')).toBeInTheDocument();
-		expect(container.querySelector('.dot2')).toBeInTheDocument();
+	it('should have expected button label and fire onRemove handler', () => {
```

Comment:
A new issue was entered to revisit the spinner and its tests. #35 

fine for this PR